### DISCUSSION TITLE
Remove style from in-line citations from revealjs output

### DISCUSSION THREAD

**Discussion by inigourrestarazu** [quarto-dev/quarto-cli#10048] [Q&A] [answered]
### Description

Hi!

I am making a presentation with revealjs. I recently discovered that citations can be included in revealjs output and that references appear at the end of the presentation with the usual pandoc syntax: `[@reference]`. Yet, the output highlights the reference in a different format. I guess it's because they are treated as hyperlinks. E.g.:

```
## Hiru bokal sudurkari(tu) iturri zubereraz [@hualde2003: 31. or.]
```

<img width="648" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/94745151/32d2c711-73cc-4e11-a62b-95ead78b3c2e">

```
**Transfonologizazioa** Larraineko euskaraz? [adib. @kirby2022]
```

<img width="692" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/94745151/061dafbe-542c-45ed-a532-514ff9832b9a">

Of course, this is not desirable, because it draws the attention to the reference. Is there any way to turn this "off" (while keeping the hyperlinks)?

I tried [@hualde2003: 31. or.]{style="color:black;". The format of the reference changed to "Hualde (1993: 31. or.)", instead of the desired "(Hualde 1993: 31. or.)", and the date remained highlighted.

Quarto version:: 1.4.555

Thank you so, so much :)

**Comment by mcanouil** [comment]
Use your favourite browser inspect/developer mode, look at the (S)CSS rule controlling this, change it.

See <https://quarto.org/docs/presentations/revealjs/themes.html#customizing-themes>.

**Comment by inigourrestarazu** [comment, marked as answer]
Yay!!!!

Including this in the scss solved it!!!

```
.reveal .citation a {
  color: inherit;
}
```

Thanks :)

**Reply by mcanouil** [reply]
Could you format your code using markdown code blocks?
